src/status_im/contexts/wallet/db.cljs

```diff
@@ -0,0 +1,9 @@
+(ns status-im.contexts.wallet.db
```

Comment:
Created a ns for wallet-related app-db defaults.

Reply:
Hey @smohamedjavid 

I'm a bit against this abstraction, I think it's too soon to add it and probably we don't need it.

I don't see a benefit here from marking this as `:const`, probably we are over `:const`-ing stuff? I've had problems while reloading some namespaces in the REPL because consts aren't allowed to re-evaluate :thinking: but probably it's just me.

The main reason I'm against is that it hides the keys used, we must navigate to this namespace to know the actual values, this might lead to bugs or unexpected cases since it's easier to forget what are the involved keys, for example, while writing a new event that involves updating an existing map. I believe keys used in event/sub handlers shouldn't be decoupled since they give the whole context and our db is not small.

Additionally, the values written here are totally derived from `status-im.constants`, I think we could just add a `def` (or even `let`) in the namespace/function we want to use it.

cc-ing @J-Son89 to get another perspective.

Reply:
>  It's just that it will be repetitive across the codebase. and any changes require updates on all places (if not done properly, may introduce bugs as well).

I know, I've had this same problem in different apps I've built using re-frame. I use an IntelliJ feature that returns all the usages of a keyword, but I know, it may still create bugs :thinking: 

> It's just that it will be repetitive across the codebase. 

Sometimes I think we over abstract, I've seen components with a `constants` ns only containing some vars being used in its `style` ns and they could be defined in the same `style` ns.

 I've also seen a component being spread across many files, each file containing a piece for the main component, I know that's a pattern in JS, but in CLJS we don't always do that.
 
 I prefer to abstract when I clearly see theres a use that will be repeated too constatly, and when the abstraction really makes things easier.
 
 In this case, if you believe abstracting makes things easier, go ahead please.